packages/flutter/lib/src/rendering/sliver_group.dart

```diff
@@ -293,46 +298,37 @@ class RenderSliverMainAxisGroup extends RenderSliver with ContainerRenderObjectM
       }());
     }
 
-    final double totalScrollExtent = offset;
-    offset = 0.0;
-    child = firstChild;
-    // Second pass to correct out of bound paintOffsets.
```

Comment:
It looks like this comment was removed… should it maybe be re-added or is it inaccurate?

Reply:
I have added a test for this.

Reply:
Added some comments.